src/renderer/components/CreatableSelect.tsx

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable react/jsx-props-no-spreading */
+
+import { CSSProperties } from 'react';
+import { StylesConfig } from 'react-select';
+import OriginalCreatableSelect from 'react-select/creatable';
+
+import sassVariables from '../styles/variables.module.scss';
+
+export default function CreatableSelect({
+  controlStyles = {},
```

Comment:
The wrapper will now ignore the 'styles' prop. If we need to override any other part of the `styles` prop passed to the underlying component, we'll need to add separate props for each. Ideally we'd unpack the `styles` prop here and merge our custom styles with what was passed to the wrapper, but this won't we trivial to implement either.

In hindsight, I think this solution (which I suggested) is much more cumbersome than what previously had (overriding with Sass). WDYT?

Please don't take my suggestions as "must haves". I was worried when I saw how much custom styling for the internal structure of this component we are doing. A quick search revealed that `CreatableSelect` has some mechanism for custom styling so I suggested it without diving into it. Use your own judgement and if my advice leads to a lot of work with little benefit - don't take it. We can always have a conversation here or in person to discuss this kind of changes.

That's some thoughts for future. Let's leave it as is and get this PR merged already; I don't think it's worthwhile to spend more time on it.